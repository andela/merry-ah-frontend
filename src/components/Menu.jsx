import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';
import Search from './Search';
import Logo from './Logo';

export default class Menu extends Component {
  render() {
    return (
      <nav id="nav" class="navbar navbar-expand-lg bg-light">
        <Logo/>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarSupportedContent">
        <Search/>
         <ul class="nav navbar-nav mr-auto col-lg-2 float-right offset-1">
            <li class="nav-item">
              <NavLink to="/login" >Login</NavLink>
            </li>
            <li class="nav-item offset-1">
              <NavLink to="/register" >Register</NavLink>
            </li>
          </ul>
        </div>
      </nav>
    )
  }
}
