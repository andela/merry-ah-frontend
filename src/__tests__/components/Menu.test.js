/* eslint-disable import/no-unresolved */
import React from 'react';
import { shallow } from 'enzyme';
import Menu from '../../components/Menu';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<Menu />);
});
describe('Menu Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
