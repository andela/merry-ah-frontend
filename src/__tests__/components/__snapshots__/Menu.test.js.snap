// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Component should match snapshot 1`] = `
<nav
  className="navbar navbar-expand-lg bg-light"
  id="nav"
>
  <Logo />
  <button
    aria-controls="navbarSupportedContent"
    aria-expanded="false"
    aria-label="Toggle navigation"
    className="navbar-toggler pull-right"
    data-target="#navbarSupportedContent"
    data-toggle="collapse"
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    >
      <i
        className="fa fa-bars"
      >
         
      </i>
    </span>
  </button>
  <div
    className="collapse navbar-collapse"
    id="navbarSupportedContent"
  >
    <Search />
    <ul
      className="navbar-nav mr-auto offset-1"
    >
      <li
        className="nav-item offset-md-5"
      >
        <NavLink
          activeClassName="active"
          aria-current="page"
          to="/login"
        >
          Login
        </NavLink>
      </li>
      <li
        className="nav-item offset-md-5"
      >
        <NavLink
          activeClassName="active"
          aria-current="page"
          to="/register"
        >
          Register
        </NavLink>
      </li>
    </ul>
  </div>
</nav>
`;
